recipe/meta.yaml

```diff
@@ -1,27 +1,29 @@
-{% set version = "0.34.2" %}
+{% set version = "0.35.0" %}
 
 package:
   name: wheel
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/w/wheel/wheel-{{ version }}.tar.gz
-  sha256: 8788e9155fe14f54164c1b9eb0a319d98ef02c160725587ad60f14ddc57b6f96
+  sha256: 625e645aae0706ad09690bded523196a065fbcf9afcdb2d713b59e01ef8699c0
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: python setup.py install --single-version-externally-managed --record=record.txt
   entry_points:
     - wheel = wheel.cli:main
 
 requirements:
   host:
     - python
+    - setuptools >=42
     - setuptools_scm >=3.4
+    - toml
   run:
     - python
-    - setuptools
+    - packaging >=20.2
```

Comment:
Updated based on this

https://github.com/pypa/wheel/blob/4253bbb865823101d1a1757d12239fc74f49c4e7/setup.cfg#L37